lib/StripeMethod.js

```diff
@@ -115,7 +115,22 @@ function stripeMethod(spec) {
         }
       }
 
-      self._request(requestMethod, requestPath, data, opts.auth, options, requestCallback);
+      var isStream = utils.checkForStream(data);
+
+      if (isStream) {
+        var bufferArray = [];
+        data.file.data
+          .on('data', function(line) {
+            bufferArray.push(line);
+          })
+          .on('end', function() {
+            var dataBuffered = Object.assign({}, data);
+            dataBuffered.file.data = Buffer.concat(bufferArray);
```

Comment:
I believe this is fine, but would you mind adding `var Buffer = require('safe-buffer').Buffer;` at the top of the file? That way if we add more code that uses `Buffer` in this file it will use the safe version.

Reply:
@ob-stripe thanks for the quick response and catching my oversight on `safe-buffer`! I noticed `safe-buffer` in the MultipartDataGenerator.js file, but totally forgot about requiring it here. I'll update the code asap and keep a look out for any other feedback from @jlomas-stripe @rattrayalex-stripe .